core/storage/topic/gae_models.py

```diff
@@ -52,7 +52,7 @@ class TopicModel(base_models.VersionedModel):
     # The canonical name of the topic, created by making `name` lowercase.
     canonical_name = ndb.StringProperty(required=True, indexed=True)
     # The abbreviated name of the topic.
-    abbreviated_name = ndb.StringProperty(indexed=True)
+    abbreviated_name = ndb.StringProperty(indexed=True, default="")
```

Comment:
By default, we use single quotes in Python.

Reply:
Done.